virtool/samples/api.py

```diff
@@ -982,7 +982,7 @@ async def upload_cache_artifact(req):
     errors = virtool.uploads.utils.naive_validator(req)
 
     if errors:
-        return invalid_query(errors)
+        return InvalidQuery(errors)
```

Comment:
```suggestion
        raise InvalidQuery(errors)
```
This should be `raise` right?